docs/runs.md

```diff
@@ -0,0 +1,227 @@
+<!--
+---
+linkTitle: "Runs"
+weight: 2
+---
+-->
+
+# `Run`s
+
+- [Overview](#overview)
+- [Configuring a `Run`](#configuring-a-run)
+  - [Specifying the target Custom Task](#specifying-the-target-custom-task)
+  - [Specifying `Parameters`](#specifying-parameters)
+- [Monitoring execution status](#monitoring-execution-status)
+  - [Monitoring `Results`](#monitoring-results)
+- [Code examples](#code-examples)
+  - [Example `Run` with a referenced custom task](#example-run-with-a-referenced-custom-task)
+  - [Example `Run` with an unnamed custom task](#example-run-with-an-unnamed-custom-task)
+  - [Example of specifying parameters](#example-of-specifying-parameters)
+
+# Overview
+
+A `Run` allows you to instantiate and execute a [Custom
+Task](https://github.com/tektoncd/community/blob/master/teps/0002-custom-tasks.md),
```

Comment:
prolly makes sense to link to the TEP as we develop this but once the feature is ready to use it'd in important to have enough info in the docs on how this works that we don't direct ppl toward the proposal to understand how to use it

Reply:
Makes sense. As we add more functionality I'll make sure the docs stay up to date.